development/docker-compose.yml

```diff
@@ -3,14 +3,19 @@ version: '3'
 services:
     postgres:
         image: postgres:12.2
+        container_name: hackathon-template_postgres
```

Comment:
can you make this `newhacks-site-2021_postgres`

Reply:
The reason we do this, by the way, is so that the event sites and the template have different docker container names, otherwise trying to start the containers for newhacks will just start them for the template (which have slightly different schemas).

A better solution would be to eventually move the docker-compose.yml file up a directory so containers inherit their names from the repository folder (right now they default to being called `development`), but that's a larger discussion for the template repo.